.github/workflows/pr-windows.yaml

```diff
@@ -0,0 +1,21 @@
+name: "PR Windows"
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: windows-2022
+    if: "!contains(github.event.head_commit.message, 'ci skip')"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Build
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          bazelisk --output_user_root=D:/0 --windows_enable_symlinks build --config=untrusted-ci-windows --remote_header=x-buildbuddy-api-key=${{ secrets.BUILDBUDDY_ORG_API_KEY }} //enterprise/server/cmd/executor:executor
```

Comment:
nit: can you add a comment about why `--windows_enable_symlinks` is enabled?

also maybe add a comment about why we're setting `--output_user_root=D:/0`